STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.cs

```diff
@@ -40,19 +42,35 @@ internal HttpExchangeService(IHttpBroker httpClientBroker)
                 httpExchange,
                 httpResponseMessage);
         });
-
+        
         private static HttpRequestMessage MapToHttpRequest(
-            HttpExchangeRequest httpExchangeRequest)
+            HttpExchangeRequest httpExchangeRequest,
+            HttpMethod defaultHttpMethod)
         {
             Uri baseAddress = new Uri(httpExchangeRequest.BaseAddress);
             string relativeUrl = httpExchangeRequest.RelativeUrl;
-
+            var httpMethod = GetHttpMethod(httpExchangeRequest, defaultHttpMethod);
```

Comment:
HttpMethod httpMethod =
because type is not clear from right side

Reply:
Treated